SpatialGDK/Documentation/content/get-started/starter-template/get-started-template-cloud.md

```diff
@@ -1,148 +1,226 @@
-
-Your deployment(s) won’t launch instantly. A console window is displayed where you can see their progress.
 
-When your deployment(s) have launched, you can open the [Console](https://console.improbable.io/) to view them.
-
-<%(#Expandable title="Cloud workflow reference diagram")%>
+To launch a cloud deployment from Unreal Editor:
+
+1. On the GDK toolbar, select **Deploy**.<br/><br/>
+![GDK toolbar "Deploy" button]({{assetRoot}}assets/screen-grabs/toolbar/gdk-toolbar-deploy.png)<br/>
+_Image: The GDK toolbar's **Deploy** button_<br/></br>
+This opens the Cloud Deployment dialog box:</br></br>
+<%(Lightbox title ="Cloud Deployment" image="{{assetRoot}}assets/screen-grabs/starter-template-deployment-launch.png")%>
+<br/> _Image: The Cloud Deployment dialog box_<br/></br>
+
+1. In the dialog box, enter the following information in the relevant fields:</br></br>
+  * **Project Name**: The SpatialOS cloud project name you associated with your Unreal project in [Step 1](#step-1-associate-your-game-with-a-spatialos-cloud-project), above. </br>
+(In the example it was `beta_yankee_hawaii_621`.) </br></br>
+  * **Assembly Name**: The name you gave the assembly you uploaded in [Step 3](#step-3-upload-built-out-workers), above.
+</br>
+(In the example it was `myassembly`.)</br></br>
+  * **Deployment Name**: You need to create a name for your deployment and add it here, this name labels the deployment in the Console.
+</br>
+(In the example dialog box shown, it's `mydeployment` but it can be whatever you choose and contain any alpha-numeric characters.)</br></br>
+  * **Launch Config File**: The absolute filepath the your Unreal project's `.json` launch configuration file from `C:/`. </br>
+  This is: `C:/...<filepath>.../<ProjectRoot>/spatial/one_worker_test.json`.
+  <%(#Expandable title="What's the launch configuration file?")%>
+This `.json` file contains the configuration parameters for starting a deployment, including:</br> * the game template - defines the compute resources your deployment needs (see the documentation on [game templates and pricing](https://docs.improbable.io/reference/latest/shared/pricing-and-support/pricing-details#introduction-to-game-templates)).</br> * `dimensionsInWorldUnits` - defines the size of your SpatialOS game world in X and Y.</br> * worker types - lists the worker type names you have set up for your project. For this project; an Unreal server-worker `UnrealWorker` and an Unreal client-worker `UnrealClient`.
+</br></br>
+This project comes with a ready-made launch configuration file `one_worker_test.json`. Note that you can call your launch configuration file any name you choose. </br>
+You can find out more in the SpatialOS Worker SDK documentation: [launch confguration file](https://docs.improbable.io/reference/14.0/shared/flexible-project-layout/reference/launch-configuration) (but note that this may contain details on parameters not relevant to Unreal-developed projects.)
```

Comment:
```suggestion
You can find out more in the SpatialOS Worker SDK documentation: [launch confguration file](https://docs.improbable.io/reference/14.0/shared/flexible-project-layout/reference/launch-configuration) (but note that this may contain details on parameters not relevant to Unreal-developed projects).
```